database/index.js

```diff
@@ -0,0 +1,39 @@
+const Sequelize = require('sequelize');
+const sampleData = require('./dataGenerator.js');
+
+const sequelize = new Sequelize('yumnoms', 'root', '', {
+  host: 'localhost',
+  dialect: 'mysql',
+});
+
+const Review = sequelize.define('review', {
+  review_id: { type: Sequelize.STRING },
+  business_id: { type: Sequelize.STRING },
+  user: { type: Sequelize.STRING },
+  stars: { type: Sequelize.INTEGER },
+  date: { type: Sequelize.DATEONLY },
+  text: { type: Sequelize.TEXT },
+  useful: { type: Sequelize.INTEGER },
+  funny: { type: Sequelize.INTEGER },
+  cool: { type: Sequelize.INTEGER },
+});
+
+const Business = sequelize.define('business', {
+  business_id: { type: Sequelize.STRING },
+  name: { type: Sequelize.STRING },
+});
+
+const Photo = sequelize.define('photo', {
+  photo_id: { type: Sequelize.STRING },
+  review_id: { type: Sequelize.STRING },
+  caption: { type: Sequelize.STRING },
+  label: { type: Sequelize.STRING },
+  imageUrl: { type: Sequelize.STRING },
+});
+
+Review.sync({ force: true, logging: false })
+  .then(() => Review.bulkCreate(sampleData.sampleData.reviews, { logging: false }))
```

Comment:
consider keeping your schema setup in a separate file from your seeding function calls. From the way it is written here, every time you require this file, Sequalize would be resyncing your database and reseeding your tables with your sample data rather than only when you run your npm script specifying to do so.